modules/manta/germline/main.nf

```diff
@@ -29,7 +30,7 @@ process MANTA_GERMLINE {
     def options_manta = target_bed ? "--exome --callRegions $target_bed" : ""
```

Comment:
IMO the `--exome` should be removed herre and put into the `modules.config` in the pipeline itself. These two variable can be used independently